packages/docs/src/scripts/helpers/polyfills.js

```diff
@@ -5,6 +5,7 @@
 // - github.com/facebook/react/issues/8379#issuecomment-264867090
 import 'core-js/fn/array/includes';
 import 'core-js/fn/array/fill';
+import 'core-js/fn/arrayfind-index';
```

Comment:
We remove this from here, right? Looks like it was moved into Tabs.jsx. This also seems to break things...

![image](https://user-images.githubusercontent.com/371943/31641166-cddf3f94-b2b0-11e7-9874-fe0ac2664f33.png)

Reply:
Yes, we should be able to remove this here, since it was added to the Tabs.jsx component file.